translations/fritzing_ru.ts

```diff
@@ -2186,17 +2186,17 @@ It is also possible to choose a connector as a ground fill seed by right-clickin
     <message>
         <location filename="../src/mainwindow/mainwindow.cpp" line="2003"/>
         <source>%1 - [%2]</source>
-        <translation>%1 - [%2]</translation>
+        <translation>%1 — [%2]</translation>
```

Comment:
Why did you replace - with — ?

Reply:
This is according to the rules of the Russian language («—» should be used between phrases; «-» cannot be preceded or followed by spaces).

Reply:
It is not used for phrases, but for separating the filename and title in the Window title. 
I looked up a random screenshot to show what I mean, see the windows title here: http://s020.radikal.ru/i708/1702/d9/cce20896172b.png . Does this rule still apply?

Reply:
Yes, it still applies in this case.